snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/event/PageView.kt

```diff
@@ -18,9 +18,7 @@ import com.snowplowanalytics.core.constants.TrackerConstants
 /**
  * A pageView event.
  * @param pageUrl The page URL.
- * @Deprecated This event has been designed for web trackers, and is not suitable for mobile apps.
```

Comment:
Maybe we can keep the comment? We do have a comment like that on the iOS tracker and it is true that page views are not suitable for mobile apps (disregarding web views), so might be a relevant info to guide users away from this in case they want standard mobile tracking? Android Studio doesn't seem to do anything about the comment.

Reply:
I'll move the comment to the main part of the docstring, instead of using the Deprecated annotation